backend/migrations/20230703130648_food_additive.up.sql

```diff
@@ -1,5 +1,5 @@
 -- Add up migration script here
-CREATE TYPE additive AS ENUM ('COLORANT','PRESERVING_AGENTS','ANTIOXIDANT_AGENTS','FLAVOUR_ENHANCER','PHOSPHATE','SURFACE_WAXED','SULPHUR','ARTIFICALLY_BLACKENED_OLIVES','SWEETENER','LAXATIVE_IF_OVERUSED','PHENYLALANINE','ALCOHOL','PRESSED_MEAT','GLAZING_WITH_CACAO','PRESSED_FISH');
+CREATE TYPE additive AS ENUM ('COLORANT','PRESERVING_AGENTS','ANTIOXIDANT_AGENTS','FLAVOUR_ENHANCER','PHOSPHATE','SURFACE_WAXED','SULPHUR','ARTIFICIALLY_BLACKENED_OLIVES','SWEETENER','LAXATIVE_IF_OVERUSED','PHENYLALANINE','ALCOHOL','PRESSED_MEAT','GLAZING_WITH_CACAO','PRESSED_FISH');
```

Comment:
this rename will be difficult to migrate withut loosing all data in the database...

Reply:
ok, renaming enum variants is possible. Should we do that as a migration or change thah here and run the rename in the production db manually?

Reply:
manually change if possible